CMakeLists.txt

```diff
@@ -215,6 +215,7 @@ endif()
 # HIP is always required
 find_package(hip REQUIRED PATHS /opt/rocm)
 message(STATUS "Build with HIP ${hip_VERSION}")
+option(APPEND_INTERFACE_INCLUDE_DIRECTORIES "Append include directories to compiler flags" ON)
```

Comment:
This option should be automatically set OFF when hipRTC compiler is used (i.e. when `MIOPEN_USE_HIPRTC` is set to ON). Use `set_var_to_condition` to conditionally set the default value (the technique is described at https://github.com/ROCm/MIOpen/pull/2694#discussion_r1482098759)

Reply:
Done